docs/dev/conventions/naming_conventions.md

```diff
@@ -1,11 +1,29 @@
-Casing Types:
-- [Kebab-case](https://en.wikipedia.org/wiki/Kebab_case)
-
 ## Java
 
-- In general, use the naming conventions from the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5-naming).
-- Use a `new` prefix for factory methods (e.g. `newFoo()` instead of `createFoo()`).
+- Follow the naming in: [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5-naming)
+- Use a `new` prefix for factory methods (e.g. `newFoo()` instead of `createFoo()`)
 
 ## URLs
 
-- kebab-case
+- [kebab-case](https://en.wikipedia.org/wiki/Kebab_case)
+
+
+## Database
+
+- [snake_case](https://en.wikipedia.org/wiki/Snake_case) 
+- Use singular names, eg: `user` instead of `users`
+- Use `id` for primary key column name
+- Foreign key columns are named after the table the refer to, eg: `<table_name>.id`
```

Comment:
There are some scenarios where this doesn't work:
For example with bans, we track the banned person, and the banning moderator, both of which are users.

Reply:
Good point, I think that is solved by adding a prefix word, eg:
```
banned_user_id
moderator_user_id
```

Should that be spelled out in the document, or should we emphasize that this point is more guideline rather than requirement?

Reply:
Spelling it out is probably better. If I found this odd, other people will probably find this odd as well.

Reply:
makes sense, I added f639f8f7c with some clarifications and a note to this case, let me know what you think @RoiEXLab